modular_skyrat/modules/ashwalkers/code/items/ash_tool.dm

```diff
@@ -58,3 +58,39 @@
 /datum/crafting_recipe/ash_recipe/ash_crowbar
 	name = "Ash Crowbar"
 	result = /obj/item/crowbar/ashwalker
+
+/obj/item/cursed_dagger
+	name = "cursed ash dagger"
+	desc = "A blunted dagger that seems to cause the shadows near it to tremble."
+	icon_state = "crysknife"
+	inhand_icon_state = "crysknife"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+
+/obj/item/tendril_seed
+	name = "tendril seed"
+	desc = "A horrible fleshy mass that pulse with a dark energy."
+	icon = 'modular_skyrat/modules/ashwalkers/icons/ashwalker_tools.dmi'
+	icon_state = "tendril_seed"
+
+/obj/item/tendril_seed/attack_self(mob/user, modifiers)
+	. = ..()
+	if(!isliving(user))
+		return
+	var/mob/living/living_user = user
+	to_chat(living_user, span_warning("You begin to squeeze [src]..."))
+	if(!do_after(living_user, 4 SECONDS, target = src))
+		return
+	to_chat(living_user, span_warning("[src] begins to crawl between your hand's appendages, crawling up your arm..."))
+	living_user.adjustBruteLoss(35)
+	if(!do_after(living_user, 4 SECONDS, target = src))
+		return
+	to_chat(living_user, span_warning("[src] wraps around your chest and begins to tighten... you feel an odd needle sensation..."))
+	living_user.adjustBruteLoss(35)
+	if(!do_after(living_user, 4 SECONDS, target = src))
+		return
+	to_chat(living_user, span_warning("[src] leaps from you satisfied and begins to grossly assemble itself!"))
+	var/type = pick(/obj/structure/spawner/lavaland, /obj/structure/spawner/lavaland/goliath, /obj/structure/spawner/lavaland/legion)
+	new type(user.loc)
```

Comment:
may want to give it a delay or smth

Reply:
I just want to see how this goes... it should be fine without? Not entirely opposed though to adding a delay.